core/common/src/ipc/IpcWebSocket.ts

```diff
@@ -151,25 +153,42 @@ export class IpcWebSocketBackend extends IpcWebSocket implements IpcSocketBacken
   }
 
   private async dispatch(_evt: Event, message: IpcWebSocketMessage) {
-    if (message.type !== IpcWebSocketMessageType.Invoke || !message.method)
+    if (message.type !== IpcWebSocketMessageType.Invoke)
       return;
 
-    const handler = this._handlers.get(message.channel);
-    if (!handler)
+    this._processingQueue.push(message);
+    this.processMessages();
```

Comment:
lint. Did you mean to await here?

Reply:
Yes. It doesn’t really matter for the logic here, but it would be more correct.